terasoluna-gfw-common-libraries/terasoluna-gfw-common/pom.xml

```diff
@@ -128,8 +128,8 @@
       <scope>test</scope>
```

Comment:
```xml
    <!-- == Begin BeanValidation == -->
    <dependency>
      <groupId>org.hibernate.validator</groupId>
      <artifactId>hibernate-validator</artifactId>
    </dependency>
    <!-- == End BeanValidation == -->
```

- [ ] It is correct to depend on as `test  scope`. Bacause `hibernate-validator` is not required for `@ExistInCodeList`.
If you fix it, you should add dependency to `jakarta.validation-api` as `optional` as well as `terasoluna-gfw-codepoints`, and you must add dependency to `hibernate-validator` in `terasoluna-gfw-common-dependencies`.

Reply:
fix.
a8c8e6ebe06e3015c99bca8390265a511900a34d

Reply:
It seems better to add blank lines before and after.

Reply:
fix.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1039/commits/21cc1dc91e14f255f5a4920e11cff4367b876752